shared/README.md

```diff
@@ -4,28 +4,67 @@
 ## Requirements
 
 - Azure Subscription
-- Service Principal
 - [Terraform](https://www.terraform.io/downloads.html)
 
+Fill the values required below in backend.tfvars to initialize Azure backend for Terraform, using the information [below](###Configure-Terraform-to-Store-State-Data-in-Azure)
+
+- Storage Account Name
+- Storage Account Access Key
+- Storage Account Container Name
+- Key Name to store for Terraform State for Test Environment
+
+
 ## Resources
 
-The following respources will be deployed
+The following resources will be deployed:
 - Azure Resource Group
 - Azure KeyVault 
 
 ## Deployment
 
-1. Authenticate using your Azure Principal or an Azure account with privileges to deploy resource groups.
+### Authenticate using your Azure Principal or an Azure account with privileges to deploy resource groups.
 
 ``` bash
 $ az login
 ```
 
-2. Execute the following commands:
+### Configure Terraform to Store State Data in Azure
+
+Terraform records the information about what is created in a [Terraform state file](https://www.terraform.io/docs/state/) after it finishes applying.  By default, Terraform will create a file named `terraform.tfstate` in the directory where Terraform is applied.  Terraform needs this information so that it can be loaded when we need to know the state of the cluster for future modifications.
```

Comment:
Instead of _cluster_, let's use language like _infrastructure_.  _Cluster_ could be confusing to the reader in the context of this project.